lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -19,6 +19,18 @@ def initialize(vmpooler_hosts, options)
       @options = options
       @logger = options[:logger]
       @hosts = vmpooler_hosts
+      @credentials = load_credentials(@options[:dot_fog])
```

Comment:
Is there somewhere that specifies which hypervisors need access to a credentials file?  I didn't see it anywhere in documentation so I'm not sure where to update?